game/main.py

```
@@ -0,0 +1,143 @@
+    glBegin(GL_LINES)
+    for edge in edges:
+        for vertex in edge:
+            glVertex3fv(vertices[vertex])
+    glEnd()
+    glPopMatrix()
+
+
+def main():
+    pygame.init()
+    pygame.mixer.init()
+    screen = pygame.display.set_mode((800, 600), DOUBLEBUF | OPENGL)
```

Ruff (0.11.9)

Local variable `screen` is assigned to but never used

Remove assignment to unused variable `screen`

(F841)